src/br/com/sunty/models/course/Course.java

```diff
@@ -0,0 +1,131 @@
+package br.com.sunty.models.course;
+
+import br.com.sunty.models.Category.Category;
+import br.com.sunty.models.instructor.Instructor;
+
+import static br.com.sunty.models.validations.Validation.*;
+
+public class Course {
+
+    private Long id;
+    private String name;
+    private String urlCode;
+    private Integer timeForFinish;
+    private CourseVisibility courseVisibility;
```

Comment:
Já estamos na classe `Course`, não há a necessidade do atributo ter `course` como prefixo.

```suggestion
    private CourseVisibility visibility;
```